### ISSUE TITLE
Mod_wsgi.so not found in appachr modules 

### ISSUE THREAD

**Issue by pavan2sunny** [GrahamDumpleton/mod_wsgi#319]
Im trying this pip install mod-wsgi in windows10 x64 it installs successfully but its not building .so file in appache modules folder

**Comment by GrahamDumpleton** [comment]
That is correct, it isn't meant to. Things get installed under the Python installation or virtual environment. Once installed, run:

```
mod_wsgi-express module-config
```

and it will output what you need to add to the Apache configuration. The mod_wsgi module will usually have a ``.pyd`` extension. Use whatever ``module-config`` displays.

Use Python 3.6 and avoid Python 2.7 it at all possible. Python 2.7 often doesn't work for unknown reasons. The extension ``module-config`` generates can also be wrong, showing ``None`` instead of ``.pyd``.

**Comment by pavan2sunny** [comment]
i copied the mod_wsgi_express output and restarted the appache server which causes the error

**Comment by pavan2sunny** [comment]
LoadFile "c:/programdata/anaconda3/python36.dll"
LoadModule wsgi_module "c:/projects/streak/lib/site-packages/mod_wsgi/server/mod_wsgi.cp36-win_amd64.pyd"
WSGIPythonHome "c:/projects/streak"
this is what i have copied in httpd.cnf

**Comment by GrahamDumpleton** [comment]
I will take it that since you closed the issue, problem is solved.
